packages/UI/src/styles/theme.ts

```diff
@@ -8,7 +8,7 @@ const theme = {
   },
   colors: {
     BLUE: '#3A7CA5',
-    BLUE_100: '#75A3C0',
+    BLUE_100: '#7A87C7',
```

Comment:
changes to BLUE_100, YELLOW_100, ORANGE_100, SUCCESS_100 and this file as a whole need to be reviewed by design team before merging

Reply:
Thanks Mohammed. 

@Nouri-Anouar can you take a look at the changes here, or let us know if there is someone else who can help? Thanks!

Reply:
@dbradham it appears this file is outdated and does not align with our design system.

@Enjoy2Live can we have colors from the design tokens here?
 [Color Pallete](https://www.figma.com/file/EwzuhhvTulvFRMvhTD5VAh/DL-Universal-Design-System?type=design&node-id=11487-35013&mode=design&t=oteG1Aryo17ME3pK-4)

Reply:
@Enjoy2Live @Nouri-Anouar this seems out of scope for Hindu's change. Neither the old color, nor the color she is updating to are in the current design system you shared here.

CC: @JulieMass

Reply:
@dbradham true and for that I'd recommend that your team to use tailwind for cases like these because anything related to styled components is considered outdated/deprecated, for the now I think you should use the token `Blue-blue-100` as tailwind className.
Example: `bg-Blue-blue-100` applies #75A3C0 as background color

@Nouri-Anouar looking at tokens studio I'm still seeing `Blue-blue-100` color token while it's not showing in the color pallete you linked, make sure to update either to match the other so we avoid confusions like these.

Reply:
thanks for the PR Anouar, this makes a ton of sense now.